fb_dnsmasq/recipes/default.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Cookbook Name:: fb_dnsmasq
+# Recipe:: default
+#
+# Copyright (c) 2016-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+package 'dnsmasq' do
+  action :upgrade
+end
+
+template '/etc/ethers' do
```

Comment:
oh this seems... *terrible*. /etc/ethers is a real thing, and while having dnsmasq read it is one thing, this cookbook CERTAINLY shouldn't write it. Feel free to make an fb_ethers akin to fb_hosts...  but certainly just because I don't want dnsmasq to USE /etc/ethers doesn't mean I want to DELETE it. other things use it